### DISCUSSION TITLE
Clash with skilion?

### DISCUSSION THREAD

**Discussion by SWBZA** [abraunegg/onedrive#1247] [Q&A] [answered]
Hi.

My onedrive on Debian has stopped syncing.  I thought I had the skilion version installed, and if I do:

```
onedrive --version                                                                    
onedrive v1.1.3
```
... it looks like it is.   Yet, when I try to upgrade, I get:

```sudo apt install onedrive                                                             
[sudo] password for schalk: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
onedrive is already the newest version (2.4.9-1).
```
... which would seem to suggest I have the abraunegg version.  

It would seem I have both applications installed.  Any help for resolving this clash?  Any pointers much appreciated.

Kind regards
Schalk

**Comment by abraunegg** [comment, marked as answer]
@SWBZA 
You need to uninstall all package versions. Refer to your distribution documentation to remove packages & any stale dependencies. Then perform a full file system walk for any binary named 'onedrive'. Once all references are removed, reinstall the right package for your platform.

**Reply by SWBZA** [reply]
Thank you, that seems to have done the trick.
